app-crypt/cfssl/cfssl-1.4.1-r1.ebuild

```diff
@@ -30,4 +28,5 @@ src_compile() {
 src_install() {
 	dobin bin/*
 	dodoc CHANGELOG README.md
+	mv -iv "${ED}"/usr/bin/mkbundle{,.cfssl}
```

Comment:
We may want to add a pkg_postinst message informing the user about the rename in case they are confused by upstream docs.